tests/forte/data/data_store_test.py

```diff
@@ -516,6 +517,26 @@ def test_add_annotation_raw(self):
         self.assertEqual(len(DataStore._type_attributes), 3)
         self.assertEqual(len(self.data_store._DataStore__tid_ref_dict), 8)
 
+    def test_add_link_raw(self):
+        self.data_store.add_link_raw(
+            "forte.data.ontology.top.Link", 9999, 1234567
+        )
+        num_link = len(
+            self.data_store._DataStore__elements["forte.data.ontology.top.Link"]
+        )
+        self.assertEqual(num_link, 2)
```

Comment:
should probably test whether the `tid` is assigned correctly?

Reply:
added the test.